node-operators/networks/compile-binary.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Compile the Binary
+description: How to compile the Moonbeam binary to run a full Parachain node, gain access to RPC endpoints, and produce blocks, for the Moonbeam Network.
+---
+
+# Compile the Moonbeam Binary
+
+![Full Node Moonbeam Banner](/images/fullnode/compile-binary-banner.png)
+
+## Introduction
+
+There are several ways to get started running a full node on the Moonbeam network. This guide goes through the process of compiling the Moonbeam binary to be run as a systemd service. For a more general overview of running nodes, or to get started with Docker, check out the [Run a Node](/node-operators/networks/full-node) page of our documentation.
```

Comment:
```suggestion
There are several ways to get started running a full node on the Moonbeam network. This guide goes through the process of compiling the Moonbeam binary from Rust source code. For a more general overview of running nodes, or to get started with Docker, check out the [Run a Node](/node-operators/networks/full-node) page of our documentation.
```

Compiling locally does allow the systemd process, but also allow much more. And also works on OSes that don't use systemd.